README.md

```diff
@@ -4,247 +4,148 @@ This backend is currently in development and is not advised for Production workf
 changes may be made without notice. We welcome your feedback as we continue to
 explore and build. The best way to share this is via our [MongoDB Community Forum](https://www.mongodb.com/community/forums/tag/python)
 
-## Install and usage
+## Install
 
 The development version of this package supports Django 5.0.x. To install it:
 
-`pip install git+https://github.com/mongodb-labs/django-mongodb-backend`
+`pip install django-mongodb-backend`
 
-### Specifying the default primary key field
+### Resources
 
-In your Django settings, you must specify that all models should use
-`ObjectIdAutoField`.
+ Check back soon as we aim to provide more links that will dive deeper into our library!
 
-You can create a new project that's configured based on these steps using a
-project template:
+* [Developer Notes](DEV_NOTES.md)
 
-```bash
-$ django-admin startproject mysite --template https://github.com/mongodb-labs/django-mongodb-project/archive/refs/heads/5.0.x.zip
-```
-(where "5.0" matches the version of Django that you're using.)
-
-This template includes the following line in `settings.py`:
-
-```python
-DEFAULT_AUTO_FIELD = "django_mongodb_backend.fields.ObjectIdAutoField"
-```
-
-But this setting won't override any apps that have an `AppConfig` that
-specifies `default_auto_field`. For those apps, you'll need to create a custom
-`AppConfig`.
-
-For example, the project template includes `<project_name>/apps.py`:
-
-```python
-from django.contrib.admin.apps import AdminConfig
-from django.contrib.auth.apps import AuthConfig
-from django.contrib.contenttypes.apps import ContentTypesConfig
 
+## Quickstart
 
-class MongoAdminConfig(AdminConfig):
-    default_auto_field = "django_mongodb_backend.fields.ObjectIdAutoField"
+This tutorial shows you how to create a Django project, connect to a MongoDB cluster hosted on MongoDB Atlas, and interact with data in your cluster. To read more, please check our MongoDB Backend for Django tutorials page.
```

Comment:
I wouldn't call this a tutorial (it's too short and not very detailed).
Please wrap lines at 80 characters. 
This project is called "Django MongoDB Backend" not "MongoDB Backend for Django".
Where is the tutorials page?